streams/src/test/java/org/apache/kafka/streams/integration/RestoreIntegrationTest.java

```diff
@@ -107,6 +109,9 @@ private Properties props(final String applicationId) {
         return streamsConfiguration;
     }
 
+    @Rule
+    public TestName name = new TestName();
```

Comment:
the changelog topics are created by ```BeforeClass``` so it seems we need to add ```Before``` to create changelog for different *method name*.